infiniband/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,79 @@
+name: Infiniband
+files:
+  - name: infiniband.yaml
+    example_name: conf.yaml.example
+    options:
+      - template: init_config
+        options:
+          - template: init_config/default
+      - template: instances
+        options:
+          - name: infiniband_path
+            description: |
+              The path to the infiniband directory.
+            value:
+              example: "/sys/class/infiniband"
+              type: string
+              display_default: "/sys/class/infiniband"
+          - name: exclude_devices
+            description: |
+              A list of devices to exclude from the check. Devices are located in the infiniband directory. 
+              The devices are located default is /sys/class/infiniband.
+            value:
+              example:
+                - "mlx5_0"
+                - "efa0" 
+                - "ib1"
```

Comment:
would it make sense to allow regexes here?

Reply:
Can be a feature request.